docs/DeveloperGuide.md

```diff
@@ -262,6 +263,50 @@ The following activity diagram summarizes what happens when a user executes a ne
     * Pros: Will result in fewer unexpected bugs since input is expected to be optional.
     * Cons: Harder to implement.
 
+### Find Contacts by Keywords
+
+#### Implementation
+
+The find mechanism is facilitated by FindCommand and FindCommandParser. It allows users to find contacts using any of 
+their contact details.
+
+#### Usage
+
+Given below is an example usage scenario of how the findCommand mechanism behaves at each step.
+
+1. The user first launches Socius. The user can see the details of his/her contacts once Socius starts.
+
+2. The user finds a new contact by one or more contact details, for example, Name and Tag.
+
+3. The user executes the command "find n/[NAME] t/[TAG]" to find people with that Name and has that Tag.
```

Comment:
Should we use `` instead of " " for `find n/[NAME] t/[TAG]`? But I'm not rlly sure how the others usually format the commands for the documentation.

Reply:
Will look into that when I edit my portion of the DG. Thanks!!